### PR TITLE
Remove Max Version limitation from Opta

### PR DESCRIPTION
**Pull request by nsarupr**
# Description
Remove Max Version limitation from Opta

# Safety checklist
* [X] This change is backwards compatible and safe to apply by existing users
* [X] This change will NOT lead to data loss
* [X] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
N/A

### CODE REVIEW

File: opta/constants.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 OPTA_DISABLE_REPORTING = "OPTA_DISABLE_REPORTING"
 TF_FILE_PATH = "main.tf.json"
 TF_PLAN_PATH = "tf.plan"
-MAX_TERRAFORM_VERSION = "1.1.0"
+# MAX_TERRAFORM_VERSION = "1.1.0"
```
Comment (RemyDeWolf) [inline opta/constants.py:45]: Should we proactively not support v2 (would be the next major version)?

File: opta/core/terraform.py [hunk 2]
```diff
@@ -117,10 +118,10 @@ def validate_version(cls) -> None:
                 f"Invalid terraform version {current_version} -- must be at least {MIN_TERRAFORM_VERSION}"
             )
 
-        if current_parsed >= version.parse(MAX_TERRAFORM_VERSION):
-            raise UserErrors(
-                f"Invalid terraform version {current_version} -- must be less than {MAX_TERRAFORM_VERSION}"
-            )
+        # if current_parsed >= version.parse(MAX_TERRAFORM_VERSION):
```
Comment (RemyDeWolf) [inline opta/core/terraform.py:121]: if we not use some code, we can remove it to stay lean.
There is always github history to find previous versions.

File: opta/constants.py (hunk 1 above)
Comment (nsarupr) [reply opta/constants.py:45]: A good point made...we should do that... let me make this `2.0.0`. That should suffice I guess?

Comment (RemyDeWolf) [reply opta/constants.py:45]: Yes

**Comment by juandiegopalomino** [comment]
what remy said. Also, never leave commented out code in prs

**Review by RemyDeWolf — APPROVED** [review]

**Pull request merged**